frontend/src/app/patients/ManagePatients.tsx

```diff
@@ -88,14 +87,12 @@ const ManagePatients = ({ activeFacilityId }: Props) => {
             <div className="usa-card__header">
               <h2> All {PATIENT_TERM_PLURAL_CAP}</h2>
               <NavLink
+                className="usa-button usa-button--outline"
                 to={`/add-patient/?facility=${activeFacilityId}`}
                 id="add-patient-button"
               >
-                <Button
-                  outline
-                  icon={"plus"}
-                  label={` New ${PATIENT_TERM_CAP}`}
-                />
+                <FontAwesomeIcon icon="plus" />
```

Comment:
The button here was just being used to get a "look" so it was better to apply button styles to the link directly.